cmd/bucket-handlers-listobjects.go

```diff
@@ -166,15 +176,24 @@ func (api objectAPIHandlers) ListObjectsV1Handler(w http.ResponseWriter, r *http
 	}
 
 	for i := range listObjectsInfo.Objects {
-		if listObjectsInfo.Objects[i].IsEncrypted() {
+		var actualSize int64
+		if isCompressed(listObjectsInfo.Objects[i].UserDefined) {
```

Comment:
Why not have this like IsEncrypted() like function? and also DecompressedSize() equivalent of DecryptedSize() ?